src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/util/DatesTest.kt

```diff
@@ -101,4 +104,18 @@ class DatesTest {
     assertThat(a countOverlappingDays b).isZero()
     assertThat(b countOverlappingDays a).isZero()
   }
+
+  @Test
+  fun `toUiFormat formats a LocalDate correctly`() {
+    val date = LocalDate.parse("2024-01-01")
+
+    assertThat(date.toUiFormat()).isEqualTo("Monday 1 January 2024")
+  }
+
+  @Test
+  fun `toUiFormattedTime formats a LocalDate correctly`() {
+    val dateTime = OffsetDateTime.parse("2024-01-01T11:15:00Z")
+
+    assertThat(dateTime.toUiFormattedTime()).isEqualTo("11am")
```

Comment:
Should this maybe be toUiFormattedHourOfDay, given it's lobbing off the 15 minutes? (I guess that's deliberate)